Tests/Unit/Content/ContentTypeResolver/ContactSelectionResolverTest.php

```diff
@@ -0,0 +1,232 @@
+use Sulu\Bundle\HeadlessBundle\Content\ContentTypeResolver\ContactSelectionResolver;
+use Sulu\Bundle\HeadlessBundle\Content\ContentView;
+use Sulu\Bundle\HeadlessBundle\Content\Serializer\ContactSerializerInterface;
+use Sulu\Component\Content\Compat\PropertyInterface;
+
+class ContactSelectionResolverTest extends TestCase
+{
+    /**
+     * @var ContactManager|ObjectProphecy
+     */
+    private $contactManager;
+
+    /**
+     * @var ContactSerializerInterface|ObjectProphecy
+     */
+    private $contactSerializer;
+
+    /**
+     * @var ContactSelectionResolver
+     */
+    private $contactSelectionResolver;
+
+    protected function setUp(): void
+    {
+        $this->contactManager = $this->prophesize(ContactManager::class);
+        $this->contactSerializer = $this->prophesize(ContactSerializerInterface::class);
+
+        $this->contactSelectionResolver = new ContactSelectionResolver(
+            $this->contactManager->reveal(),
+            $this->contactSerializer->reveal()
+        );
+    }
+
+    public function testGetContentType(): void
+    {
+        self::assertSame('contact_selection', $this->contactSelectionResolver::getContentType());
+    }
+
+    public function testResolveWithOneContact(): void
+    {
+        $locale = 'en';
+
+        $contact = $this->prophesize(ContactInterface::class);
+        $apiContact = $this->prophesize(Contact::class);
+        $apiContact->getEntity()->willReturn($contact->reveal());
+
+        $data = [2];
+
+        $this->contactManager->getByIds($data, $locale)->willReturn([$apiContact->reveal()]);
+        $this->contactSerializer->serialize($contact, $locale, Argument::type(SerializationContext::class))->willReturn(
+            [
+                'id' => 2,
+                'firstName' => 'John',
+                'lastName' => 'Doe',
+                'fullName' => 'John Doe',
+                'title' => 'fancyTitle',
+                'position' => 'CEO',
+                'avatar' => [
+                    'id' => 2,
+                    'formatUri' => '/media/2/{format}/media-2.jpg?v=1-0',
+                ],
+            ]
+        );
+
+        $property = $this->prophesize(PropertyInterface::class);
+        $result = $this->contactSelectionResolver->resolve($data, $property->reveal(), $locale);
+
+        $this->assertInstanceOf(ContentView::class, $result);
+        $this->assertSame(
+            [
+                [
+                    'id' => 2,
+                    'firstName' => 'John',
+                    'lastName' => 'Doe',
+                    'fullName' => 'John Doe',
+                    'title' => 'fancyTitle',
+                    'position' => 'CEO',
+                    'avatar' => [
+                        'id' => 2,
+                        'formatUri' => '/media/2/{format}/media-2.jpg?v=1-0',
+                    ],
+                ],
+            ],
+            $result->getContent()
+        );
+
+        $this->assertSame(
+            [
```

Comment:
We should be consistent here to the other multi selection and not nest again into an array.

Reply:
Add it :)